### DISCUSSION TITLE
the alisit can not be installed

### DISCUSSION THREAD

**Discussion by jessyrow** [AlistGo/alist#5954] [Q&A]
i have downloaded the alist file and open it then type the cmd on the search position, the fill the alist server, then click the enter. it shows that the machine refused it , why?
![cmd](https://github.com/alist-org/alist/assets/87893404/193c38e3-57b2-4359-a00b-df0e2e80919e)

**Comment by qtnull** [comment]
Alist was able to start up correctly, please browse to `http://localhost:5244/` and you should be met with Alist's login page. The "error" means that it couldn't find any instance of Aria2 downloader or qBitTorrent client running and listening on your local machine, this warning can be ignored.

Alist 正常启动了，打开浏览器然后进入地址`http://localhost:5244/`就能看到 Alist 的登录界面了。上面的那些“错误”的意思只是 Alist 找不到任何正在运行的 Aria2 下载器或者 qBitTorrent 客户端。这些警告可以直接忽略掉。
